app/grants/views.py

```diff
@@ -3380,7 +3380,7 @@ def post(self, request):
         process_grant_creation_admin_email.delay(grant.pk)
 
         # record to notion for sybil-hunters
-        process_notion_db_write.delay(grant.pk)
+        process_notion_db_write
```

Comment:
We've dropped the call here, this should be right:

```suggestion
        process_notion_db_write(grant.pk)
```